api/v1beta3/auth_config_conversion.go

```diff
@@ -0,0 +1,4 @@
+package v1beta3
+
+// Hub marks this version as a conversion hub.
+func (a *AuthConfig) Hub() {}
```

Comment:
I don't know for sure how this works, but can we have 2 versions marked as conversion hub? I believe right now `v1beta2` is also marked as hub, right?